### PR TITLE
#1088 move hbase code to correct folder

### PR DESCRIPTION
**Pull request by whjiang**
[Review on Reviewable](https://reviewable.io/reviews/intel-hadoop/gearpump/1089)

### CODE REVIEW

File: project/Build.scala [hunk 1]
```diff
@@ -546,9 +546,9 @@ object Build extends sbt.Build {
     settings = commonSettings
   ) dependsOn(streaming % "test->test;compile->compile")
 
-  lazy val hbase = Project(
-    id = "gearpump-experiments-hbase",
-    base = file("experiments/hbase"),
+  lazy val external_hbase = Project(
+    id = "gearpump-external-hbase",
+    base = file("external/hbase"),
```
Comment (manuzhang) [inline project/Build.scala]: where is this directory ?

**Comment by whjiang** [comment]
the failed case is org.apache.gearpump.cluster.main.MasterWatcherSpec in daemon package. I don't think my code is relavant to this failure.

**Comment by manuzhang** [comment]
+1

**Comment by manuzhang** [comment]
please update serializers in `pipeline.conf.template` as well.

**Comment by manuzhang** [comment]
+1

**Comment by clockfly** [comment]
Please hold this merge till PR commit changes to kafka connector.

**Comment by whjiang** [comment]
@clockfly why? this PR only move the code but doesn't change the implementation. I plan to have a follow-up PR after kafka connector to support HBase connector.

**Comment by clockfly** [comment]
Hi Weihua,

Please rebase

**Comment by whjiang** [comment]
please help to review.

**Comment by huafengw** [comment]
Should the pipeline project under ```example/streaming```?
Also, since you move the pipeline project into examples, you should add the assembly settings so that it will be in the examples folder, like:
```target in assembly := baseDirectory.value.getParentFile.getParentFile / "target" / scalaVersionMajor```

**Comment by whjiang** [comment]
@huafengw updated. please review.

**Comment by huafengw** [comment]
Seems the Pipeline source code is still under examples/ , not examples/streaming

**Comment by whjiang** [comment]
@huafengw please review.

**Comment by huafengw** [comment]
+1 please rebase

**Pull request merged**